infer/src/java/jFrontend.ml

```diff
@@ -198,26 +198,18 @@ let create_icfg source_file program tenv icfg cn node =
 
 (* returns true for the set of classes that are selected to be translated in the given
    program *)
-let should_capture program package_opt source_basename node =
-  let classname = Javalib.get_name node in
-  let match_package pkg cn =
-    match JTransType.package_to_string (JBasics.cn_package cn) with
```

Comment:
I noticed that the function `JTransType.package_to_string` is not used anywhere else so it can be deleted. I will update the PR.

Reply:
Hi Jeremy. I have started removing this deadcode myself. `JProcDesc.mem_classmap` was also dead now.

Reply:
I should be able to merge this PR soon. Thanks a lot @jeremydubreil!